google/datalab/_context.py

```diff
@@ -82,6 +82,16 @@ def set_config(self, config):
     """ Set the config property for the context. """
     self._config = config
 
+  def override(self, project=None, zone=None, credentials=None, config=None):
```

Comment:
Was this supposed to come back?

Reply:
Nope; there was a bad merge. Removed it now.